subsys/nfc/ndef/ch_rec_parser.c

```diff
@@ -0,0 +1,551 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <errno.h>
+#include <logging/log.h>
+#include <net/buf.h>
+#include <sys/byteorder.h>
+
+#include <nfc/ndef/msg_parser.h>
+#include <nfc/ndef/ch_rec_parser.h>
+#include <nfc/ndef/payload_type_common.h>
+
+#define TAG_TYPE_4_NLEN_FIELD_SIZE 2
```

Comment:
I think this definition can be removed now

Reply:
Removed